lib/commands/command-install.bash

```diff
@@ -71,11 +72,38 @@ install_local_tool_versions() {
 
   local some_tools_installed
 
+  local tool_versions_path
+  tool_versions_path=$(find_tool_versions)
+
+  # Locate all the plugins installed in the system
+  local plugins_installed
   if ls "$plugins_path" &>/dev/null; then
     for plugin_path in "$plugins_path"/*; do
       local plugin_name
       plugin_name=$(basename "$plugin_path")
+      plugins_installed="$plugins_installed $plugin_name"
+    done
+    plugins_installed=$(printf "%s" "$plugins_installed" | tr " " "\n")
+  fi
+
+  if [ -z "$plugins_installed" ]; then
+    printf "Install plugins first to be able to install tools\\n"
+    exit 1
+  fi
 
+  # Locate all the plugins defined in the versions file.
+  local tools_file
+  if [ -f "$tool_versions_path" ]; then
+    tools_file=$(strip_tool_version_comments "$tool_versions_path" | cut -d ' ' -f 1)
+    for plugin_name in $tools_file; do
+      if [[ ! "${plugins_installed[*]}" =~ $plugin_name ]]; then
```

Comment:
Is this the same logic we use elsewhere in the codebase? I think this might cause problems if two plugins share matching prefixes. For example, `ag` and `age` are both existing plugins right now. With this logic, `age` could match the plugin name `ag`.

Reply:
No, that is the first occurrence of that comparison. I will turn it into an exact comparison.